debian/control

```diff
@@ -25,6 +24,9 @@ Homepage: https://github.com/intel/xpumanager
 Package: libxpum1
 Architecture: amd64
 Depends:
+  libigsc0 (>=0.9.3),
+  libze1 (>= 1.17.6),
+  libze-intel-gpu1 (>=1.3.23726),
```

Comment:
version needs to match at least the 24.35 version of libze...this is old version scheme

Reply:
Updated to the current version for now